odl/contrib/shearlab/test/operator_test.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2014-2018 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Unit tests for the ODL-shearlab integration."""
+
+import pytest
+import numpy as np
+import odl
+import odl.contrib.shearlab
+from odl.util.testutils import all_almost_equal, simple_fixture
+
+
+dtype = simple_fixture('dtype', ['float32', 'float64'])
```

Comment:
So here we have an option in dtypes (single/double precision) but in the code it seems like you only support double precision? Is this the case for the Julia implementation as well?